src/main/java/uk/co/ramp/event/AlertEventProcessor.java

```diff
@@ -115,12 +120,25 @@ public ProcessedEventResult processEvent(AlertEvent event) {
     }
   }
 
-  int timeInStatus(AlertStatus newStatus) {
-
+  int timeInStatus(AlertStatus newStatus, AlertEvent event) {
+    int time = event.time();
     MeanMax progressionData;
     switch (newStatus) {
       case TESTED_POSITIVE:
       case TESTED_NEGATIVE:
+        int conducted = statisticsRecorder.getTestsConducted(time);
+        int capacity =
+            (int) Math.ceil(properties.populationSize() * populationProperties.testCapacity());
+        boolean testsAvailable = conducted < capacity;
+        if (testsAvailable) {
+          statisticsRecorder.recordTestConducted(time);
+          progressionData = ImmutableMeanMax.builder().mean(1).max(1).build();
+        } else {
+          int testTime = findNextTest(time, capacity);
+          statisticsRecorder.recordTestDelayed(time, event.id());
```

Comment:
The statistics recorder appears to be recording things (such as event ids) which are never used.  Why is this?

Reply:
this is a hook in case they want to output information beyond the total number of delays.